Decompiler/Options.cs

```diff
@@ -43,6 +43,10 @@ class Options
             HelpText = "Verify checksums and signatures.")]
         public bool VerifyVPKChecksums { get; set; }
 
+        [Option('c', "vpk_cache", DefaultValue = false,
```

Comment:
Done.
I was using several Decompiler calls with caching enabled in my scripts since the manifest adds up, dumping files in different folders and the shorthand was handy. Plus it was fitting into the ..,d,e,f options added - I actually went backwards from _c_ to _cache_ :)